### PR TITLE
Create offline_enabled.json

### PR DESCRIPTION
**Pull request by vitaly-zdanevich**

### CODE REVIEW

**Comment by vitaly-zdanevich** [comment]
image

File: webextensions/manifest/offline_enabled.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "webextensions": {
+    "manifest": {
+      "name": {
```
Comment (wbamberg) [inline webextensions/manifest/offline_enabled.json]: Should be `offline_enabled` I think.

**Comment by vitaly-zdanevich** [comment]
@wbamberg extension developers may not know that this is Chromium-only thing. For me it was a surprise that Firefox shows warning about `offline_enabled` at the internal page of the extension. Also maybe in the future Firefox addons store will support this property to mark extension that have offline capability, like now in Chrome store.

**Comment by wbamberg** [comment]
There are a lot of Chrome-only APIs and keys that we do not document. The assumption should be: *if we don't document it, it's not supported in Firefox*. We should probably make that assumption clearer in the docs, but I'd prefer to do that than add docs for all the Chrome-specific stuff.

But this isn't really my area any more, so I'd be happy to go with whatever @irenesmith thinks.

**Comment by vitaly-zdanevich** [comment]
I thought that MDN is not only Firefox-specific :(

**Comment by wbamberg** [comment]
> I thought that MDN is not only Firefox-specific :(

MDN is not Firefox-specific. It documents Web standards. As far as WebExtensions are concerned the situation is less clear though because [the standard](https://browserext.github.io/browserext/) is not very active, and we document lots of Firefox-only additions to the standard (although it's not always clear to me how far these additions are suitable for inclusion in the standard).

But we haven't up to now documented Chrome-only APIs, and so a good assumption when reading these docs is: if it's not documented, it's not supported in Firefox (or Edge for that matter). As I said though, this isn't really my area any more, and we might decide to change this policy and start documenting all the Chrome APIs. I'm just talking about what we've done up to now, and what is a sensible assumption to make when reading the docs as they currently stand.

**Pull request merged**